__tests__/park.test.js

```diff
@@ -1,13 +1,13 @@
 const AWS = require('aws-sdk');
 const { DocumentClient } = require('aws-sdk/clients/dynamodb');
 const { REGION, ENDPOINT, TABLE_NAME } = require('./global/settings');
-const { PARKSLIST, SUBAREAS } = require('./global/data.json');
+const { FISCAL_YEAR_LOCKS } = require('./global/data.json');
 
-const parkGET = require('../lambda/park/GET/index');
-const parkPOST = require('../lambda/park/POST/index');
```

Comment:
You have removed code coverage for these two files.  These should be put back.

Reply:
Reinstated. Thanks for pointing it out.